src/index.test.js

```diff
@@ -46,6 +46,18 @@ test('render with decode=true', () => {
   )
 })
 
+test('render with referrerPolicy', () => {
+  const {getByAltText} = render(
```

Comment:
We should add a test when there is no referrer policy, and check that nothing is set on the <img>